simplq/src/styles/joinPage.module.scss

```diff
@@ -1,70 +1,69 @@
 @import 'utils';
- .message {
-	text-align: center;
+.message {
+  text-align: center;
 }
 .form {
-	display: flex;
-	flex-direction: column;
-	justify-content: space-between;
-	align-items: center;
-	max-width: 35rem;
-	margin: auto;
-	padding: 2rem;
-	&>div {
-		margin: 1rem;
-	}
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 35rem;
+  margin: auto;
+  padding: 2rem;
+  & > div {
+    margin: 1rem;
+  }
 }
 .header-bar {
-	@include background-color();
-	min-height: 8.5rem;
-	padding: 0.5rem 0 0 4.5rem;
-	display: flex;
-	flex-wrap: wrap;
-	.simplq-logo {
-		padding-top: 1.1rem;
-		z-index: 3;
-	}
-	.queue {
-		margin: auto;
-		position: absolute;
-		top: 1rem;
```

Comment:
Removing this position: absolute is my only change.

Reply:
keep all css as it is , just add 2 lines of css in .form class
position : relative;
z-index:10;

Reply:
This would probably work too, but I felt like the position: absolute wasn't needed. I didn't notice any problems after removing it. What was your reason for adding it?

Reply:
See here, I think that was needed, without that queue name isn't perfectly centered.
![image](https://user-images.githubusercontent.com/37216460/95567799-0c460700-0a41-11eb-817e-9db8434bb8c8.png)

Reply:
Okay, I see. I'll change to the z-index then.